pkg/networkservice/common/connect/server_test.go

```diff
@@ -59,17 +61,19 @@ func TestConnectServer_Request(t *testing.T) {
 	serverNext := new(captureServer)
 	serverClient := new(captureServer)
 
+	chainCtx, chainCancel := context.WithCancel(context.Background())
```

Comment:
Do we actually need it? Closing all networkservice connection should mean closing all gRPC connections even without context cancel.

Reply:
You are correct.  Fixed :)